modules/prebidServerBidAdapter/index.js

```diff
@@ -685,23 +679,24 @@ Object.assign(ORTB2.prototype, {
       }
 
       if (nativeAssets) {
+        const ortbRequest = deepAccess(nativeParams, 'ortb');
         try {
           mediaTypes['native'] = {
-            request: JSON.stringify({
+            request: ortbRequest ? JSON.stringify(ortbRequest) : JSON.stringify({
               // TODO: determine best way to pass these and if we allow defaults
               context: 1,
               plcmttype: 1,
               eventtrackers: [
-                {event: 1, methods: [1]}
+                { event: 1, methods: [1] }
```

Comment:
As discussed yesterday, I've left `{event: 1, methods: [1]}` because it's a more conservative choice, and also pixels are generally more accepted than the JS equivalent.